File: code/1043/1043.js
const input = require("fs").readFileSync("/dev/stdin", "utf8");
const lines = input.split(/\n|\s/);

const a = parseFloat(lines[0]);
const b = parseFloat(lines[1]);
const c = parseFloat(lines[2]);

if (a + b > c && b + c > a && c + a > b) {
  console.log(`Perimetro = ${(a + b + c).toFixed(1)}`);
} else {
  console.log(`Area = ${(((a + b) * c) / 2).toFixed(1)}`);
}

File: code/1043/1043.rb
input = gets.split(' ')

a = input[0].to_f
b = input[1].to_f
c = input[2].to_f

if a + b > c && b + c > a && c + a > b
  puts format('Perimetro = %.1f', a + b + c)
else
  puts format('Area = %.1f', ((a + b) * c) / 2)
end
